develop/dev-guide-use-stale-read.md

```diff
@@ -59,21 +68,25 @@ When we checked the latest book list again, it was found that the price of this
 5 rows in set (0.01 sec)
 ```
 
-If we do not require the use of the latest data, we can let TiDB directly return historical data that may have expired through the Stale Read feature to avoid the delay caused by data synchronization when using strong consistency read.
+If it is not necessary to use the latest data, you can query with Stale Read, which might return outdated data, to avoid the latency caused by data synchronization during a strongly consistent read.
```

Comment:
```suggestion
If it is not necessary to use the latest data, you can query with Stale Read, which might return outdated data, to avoid the latency caused by data replication during a strongly consistent read.
```